docs/compatibility.asciidoc

```diff
@@ -0,0 +1,7 @@
+[[compatibility]]
+== Compatibility
+
+Language clients are forward compatible; meaning that clients support communicating
+with greater or equal minor versions of Elasticsearch. Elasticsearch language
+clients are only backwards compatible with default distributions and without
+guarantees made.
```

Comment:
In the rest of the Client books, we added the compatibility information to the `Installation` section as a level-three sub-section:
```
[discrete]
[[compatibility]]
=== Compatibility
```
If there is no particular reason to add this info as a separate section, I would suggest moving this content under the Installation section.